src/DssSpell.sol

```diff
@@ -76,11 +76,14 @@ contract DssSpellAction is DssAction {
     // This should be modified weekly to provide a summary of the actions
     // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/c55480785534ee1f75bda65afb9761ffaebe072f/governance/votes/Executive%20vote%20-%20June%2014%2C%202023.md' -q -O - 2>/dev/null)"
     string public constant override description =
-        "2023-06-14 MakerDAO Executive Spell | Hash: 0x8d9ac29d96cec17771f198f641e3c7148aa281d6f32b276edb50084c84ab098e";
+        "2023-06-16 MakerDAO Executive Spell | Hash: TODO";
+
+    // Comma-separated list of DAO resolutions.
+    string public constant RESOLUTIONS = "QmYPbYoPjrSzBu5gt9tip78siG1gFvY8K4HTkHEFJgMmM8";
```

Comment:
I think there is probably a better name for this, but either way we normally use lower case for `string public constant` per the archives.